readme.txt

```diff
@@ -0,0 +1,36 @@
+=== User Interface Options ===
```

Comment:
The WordPress guidelines for plugin development specify that a plugin should have a file called "readme.txt" that contains the information in this file, in this format.